src/pages/QuizUi/CheckRadioRender.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { FormControlLabel, Radio, Checkbox } from '@mui/material';
+import { Box } from '@mui/system';
+import Grid from '@mui/material/Unstable_Grid2';
+
+export default function renderOptions(
+  options,
+  questionType,
+  selected,
+  setSelected,
+  isCurrentQuestionAnswered
+) {
+  return options.map((option, index) => (
+    <Grid item xs={12} key={index}>
+      <Box
+        component="div"
+        onClick={() => {
+          if (!isCurrentQuestionAnswered) {
```

Comment:
For complicated callbacks like this I'd generally suggest creating a new function in the module so that the name of the function can help readers understand what it's doing. How would you name this callback?

Reply:
Good call.  The best I could come up with is handleOptionClick, so that's what I'm going with. Thanks Eli!